src/app.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import CreateCard from './create-card'
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {}
+  }
+  componentDidMount() {
+    document.querySelector('#createFlash').addEventListener('submit', (e) => {
```

Comment:
We avoid querying the DOM when building React apps. Instead, define a method for storing a new `flashCard` and pass it as a prop to your form component.